src/Components/App/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+
+class App extends React.Component {
```

Comment:
After learning hooks today can we change app to a function.

Reply:
Converted this to a component and imported `useState`.